### PR TITLE
LLM: add new readme as first version document

### PR DESCRIPTION
**Pull request by TheaperDeng**
### 1. Why the change?
lack of document and API description.

### 2. User API changes
Nothing

### 3. Summary of the change 
https://github.com/TheaperDeng/BigDL/blob/bigdl-llm-readme/python/llm/README.md

### CODE REVIEW

File: python/llm/README.md [hunk 1]
```diff
@@ -1,47 +1,129 @@
...
-Feel free to explore different options and experiment with the llama/gptneox/bloom models using
-llm-cli!
\ No newline at end of file
+#### langchain integration
+TODO
+
+## Examples
+We prepared several examples in https://github.com/intel-analytics/BigDL/tree/main/python/llm/example
+
+## Dynamic library BOM
```
Comment (jason-dai) [inline python/llm/README.md:119]: Is this dependency on user machine?

Comment (TheaperDeng) [reply python/llm/README.md:119]: This dependency is not needed for users. But I provide these information for compatibility reason. Users who are using lower version `GLIBC` may meet compatibility issue.

The version we use to build the dynamic lib and executable files are "as-low-as-possible" to increase the compatibility.

Comment (jason-dai) [reply python/llm/README.md:119]: > This dependency is not needed for users. But I provide these information for compatibility reason. Users who are using lower version `GLIBC` may meet compatibility issue.
> 
> The version we use to build the dynamic lib and executable files are "as-low-as-possible" to increase the compatibility.

SO this is actually user system dependency - for linux users they need to install glibc v9+, yes? how about windows users?

Comment (TheaperDeng) [reply python/llm/README.md:119]: Yeah, I will add this requirement. The upgrade of GLIBC could be really tricky so most of our users will use the default GLIBC version provided by their OS. In this case, I will state we support Windows 10/11 (10 not validated), Ubuntu 20.04 or later, CentOS 7 or later (bloom could not support CentOS).

**Review by rnwang04 — APPROVED** [review]
LGTM

**Pull request merged**